easybuild/easyconfigs/p/Python/unixccompiler-Python-2.7.11.patch

```diff
@@ -0,0 +1,11 @@
+--- EasyBuildTesting/Python-2.7.9/Lib/distutils.orig/unixccompiler.py	2014-12-10 16:59:34.000000000 +0100
```

Comment:
comments in the patch is usually what we do

while you're at it, you can rename the patch to make it clear this is only relevant for h5py?